legal-api/tests/unit/models/test_business.py

```diff
@@ -473,3 +476,66 @@ def test_amalgamated_into_business_json(session, test_name, existing_business_st
         assert business_json['amalgamatedInto']['legalName'] == business.legal_name
     else:
         assert not 'amalgamatedInto' in business_json
+
+
+@pytest.mark.parametrize('test_name, legal_type, flag_on', [
+    ('GP_FLAG_ON', 'GP', True),
+    ('SP_FLAG_ON', 'SP', True),
+    ('GP_FLAG_OFF', 'GP', False),
+    ('SP_FLAG_OFF', 'SP', False),
+    ('NON_FIRM_FLAG_ON', 'BC', True),
+    ('NON_FIRM_FLAG_OFF', 'BC', False),
+])
+def test_firm_business_json(session, test_name, legal_type, flag_on):
```

Comment:
can we update this test so we can verify the scenario where there are more than 2 partners so can verify 'et al` is properly appended to the legal name?